javascript/packages/orchestrator/src/types.ts

```diff
@@ -37,6 +37,7 @@ export interface Settings {
   backchannel?: boolean; // only used in k8s at the moment, spawn a backchannel instance
   image_pull_policy?: "IfNotPresent" | "Never" | "Always";
   local_ip?: string; // ip used for expose local services (rpc/metrics/monitors)
+  bad_network_global_settings?: BadNetworkSettings;
```

Comment:
If we have this set, we need to `mv` or `copy` this config to all the node when we create the NetworkSpec.